manifests/charts/istio-operator/templates/deployment.yaml

```diff
@@ -20,6 +20,13 @@ spec:
           command:
           - operator
           - server
+          securityContext:
```

Comment:
Discovery specifies:
```
          securityContext:
            runAsUser: 1337
            runAsGroup: 1337
            runAsNonRoot: true
            capabilities:
              drop:
              - ALL
```

My understanding here is a "correct" way to do this, based on some obscure details of how PSP's mutate pods, that I am not 100% aware of. Maybe @dgn or @rlenglet can shed some light. I would expect  these to probably look the same, but the correct answer may be a mix of these two

Reply:
Agreed. It's better to specify all the fields in `securityContext` to lock down the pod and avoid surprises with PSPs. So I would specify:
```
    securityContext:
      allowPrivilegeEscalation: false
      capabilities:
        drop:
        - ALL
      privileged: false
      readOnlyRootFilesystem: true
      runAsGroup: 1337
      runAsUser: 1337
      runAsNonRoot: true
```